libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/TargetCameraMode.kt

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.navigation.dropin.component.camera
+
+sealed class TargetCameraMode {
+    object Idle : TargetCameraMode()
+    object Overview : TargetCameraMode()
+    object Following : TargetCameraMode()
+    object TransitionToOverview : TargetCameraMode()
+    object TransitionToFollowing : TargetCameraMode()
```

Comment:
These states are not utilized, are they? The type seems to reflect the **target** state, so the transition states could potentially be skipped.

Reply:
Done